proteus/mprans/SpatialTools.py

```diff
@@ -86,6 +88,60 @@ def _attachAuxiliaryVariable(self, key):
                 self.auxiliaryVariables[key] = True
             if key == 'RelaxZones':
                 self.auxiliaryVariables[key] = self.zones
+            if str(key).startswith('Gauge_'):
+                self.auxiliaryVariables[key] = [gauge]
+        elif str(key).startswith('Gauge_') \
+                and gauge not in self.auxiliaryVariables[key]:
+            self.auxiliaryVariables[key] += [gauge]
```

Comment:
The logic and indentation here looks a little suspicious. Might want to 3rd and 4th if's into elif's and terminate the blocks with else's so it's clear what happens when the key fall's through.  I may just need to look at the context more carefully.